### ISSUE TITLE
ARIA translatable attrs is missing aria-description

### ISSUE THREAD

**Issue by cookiecrook** [w3c/aria#2647]
# Describe the change
ARIA translatable attrs is missing aria-description

# Link to the version of the specification or documentation you were looking at
https://www.w3.org/TR/wai-aria-1.3/#translatable-attributes

Does the issue exists in [the editors draft](https://w3c.github.io/aria/) (the editors draft is the most recent draft of the specification)?
Yes

**Comment by cookiecrook** [comment]
Never mind... Somehow I got turned around and was looking at the published version rather than the editor's draft. This is already resolved.
